weather-forecast/src/components/InputField/InputField.tsx

```diff
@@ -28,8 +31,9 @@ const InputField = () => {
     }, [value])
 
     return (
-        <div>
-            <input className={s.input} type="text" value={value} onChange={handleChange} />
+        <div className={s.inputWrapper}>
+            <input className={cn({[s.errorInput]: error}, s.input)} type="text" value={value} onChange={handleChange} />
+            {!!error && <div className={s.error}>{error.replace(/./, error[0].toUpperCase())}</div>}
```

Comment:
I'm not sure what's going on here `error.replace(/./, error[0].toUpperCase())`.
Isn't it the same as `error[0].toUpperCase()`?